### ISSUE TITLE
Some of the editable units are not actually Editable

### ISSUE THREAD

**Issue by Ion-e** [nion-software/nionswift#1041]
![image](https://github.com/nion-software/nionswift/assets/19330696/9198bded-ee2c-4aa8-b5a6-0860ea010d42)
Inconsistency behaviour in editable units in edit boxes.

Actual behaviour:  some of the editable units are not actually editable and will default to it's original value. Please note the following screenshot shows only a couple of examples however it's more likely that more cases have the aforementioned behaviour.

Expected behaviour: edit boxes and units should behave consistently throughtout the software to avoid misinterpretations by the users. 

![image](https://github.com/nion-software/nionswift/assets/19330696/a895b1c6-00f7-4d4a-9506-029abc7a5079)

**Comment by jamesrussell216** [comment]
@Ion-e can you do us an exact list of which are the problem edit boxes? I can't tell from the screenshots alone which ones those are?

**Comment by cmeyer** [comment]
@Ion-e  Yes, please file specific issues in either `nionswift` project or `nionswift-instrumentation-kit`. Close this issue?
